env/.env.test

```diff
@@ -8,10 +8,6 @@ FLASK_DEBUG=false
 # Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
 SECRET_KEY=fake-key
 
-# Trello
-TRELLO_API_KEY=fake-key
-TRELLO_API_TOKEN=fake-token
-TRELLO_BOARD_ID=fake-id
-TRELLO_TO_DO_LIST_ID=fake-id
-TRELLO_DOING_LIST_ID=fake-id
-TRELLO_DONE_LIST_ID=fake-id
+# Mongo DB
+CONNECTION_STRING=fakemongo.com
+DATABASE_NAME=app-database
```

Comment:
I'd recommend either:
* Requiring both the collection and database names as env vars
or 
* Hard coding both

FYI the database and collection will be created automatically if they are missing